kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateExamPreview.vue

```diff
@@ -329,5 +340,12 @@
   .sortable-ghost * {
     visibility: hidden;
   }
+  .align-btn {
+    display: flex;
+    float: right;
```

Comment:
I wonder if we should consider using the [flex-direction](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-flex-direction) property rather than a `float` here.

Our use of `flex` is complicated by our continued support for IE11 but there are only a handful of [pretty specific bugs with flex in IE11](https://caniuse.com/flexbox) -- but `flex-direction` doesn't seem to be one that would cause issues.

One thing we'd need to test and be sure of, however, is that the LTR/RTL changes are reflected properly if we use `flex-direction`. Part of the motivation here is avoiding the use of `float` where, specifically, the wrapping of text isn't involved. It's not the end of the world to use it but something worth considering I think. Here is a short [blog I found interesting about float](https://css-tricks.com/is-css-float-deprecated/).

Overall I'd suggest trying it out, testing in both LTR and RTL and if it causes more problems than it's worth then the `float` should be okay.